src/algebra/ordered_group.lean

```diff
@@ -71,12 +71,18 @@ begin simp [inv_mul_cancel_left] at this, assumption end
 
 @[priority 100, to_additive]    -- see Note [lower instance priority]
 instance ordered_comm_group.to_ordered_cancel_comm_monoid (α : Type u)
-  [s : ordered_comm_group α] : ordered_cancel_comm_monoid α :=
+  [s : ordered_comm_group α] :
+  ordered_cancel_comm_monoid α :=
 { mul_left_cancel       := @mul_left_cancel α _,
   mul_right_cancel      := @mul_right_cancel α _,
   le_of_mul_le_mul_left := @ordered_comm_group.le_of_mul_le_mul_left α _,
   ..s }
 
+instance ordered_add_comm_group.has_exists_add_of_le (α : Type u)
+  [ordered_add_comm_group α] :
+  has_exists_add_of_le α :=
+⟨λ a b hab, ⟨b - a, (add_sub_cancel'_right a b).symm⟩⟩
```

Comment:
```suggestion
@[to_additive]
instance ordered_comm_group.to_has_exists_mul_of_le (α : Type u)
  [ordered_comm_group α] :
  has_exists_mul_of_le α :=
⟨λ a b hab, ⟨b / a, (mul_div_cancel'_right a b).symm⟩⟩
```
although the lemma `mul_div_cancel'_right` might be called something else, or you might have to use `b * a\inv` instead.

Reply:
I can make the mul proof work as follows: 
``instance ordered_comm_group.has_exists_mul_of_le (α : Type u)
  [ordered_comm_group α] :
  has_exists_mul_of_le α :=
⟨λ a b hab, ⟨b * a⁻¹, (mul_inv_cancel_comm_assoc a b).symm⟩⟩``
but using ``@[to_additive]`` above it gives me an error I don't understand : 

``instance ordered_comm_group.has_exists_mul_of_le (α : Type u)
  [ordered_comm_group α] :
  has_exists_mul_of_le α :=
⟨λ a b hab, ⟨b * a⁻¹, (mul_inv_cancel_comm_assoc a b).symm⟩⟩``

I'm not so familiar with the way to_additive works; is there a way to fix this, or is it acceptable to just explicitly include separate instances for additive and multiplicative?